index.js

```diff
@@ -125,7 +126,18 @@ var publicAgent = {
     if (!config.enabled) {
       return this;
     }
-    var logger = common.logger.create(config.logLevel, '@google/cloud-trace');
+
+    var logLevel = config.logLevel || 2;
```

Comment:
As in, whether it's set in stone that it should be a number rather than the log level itself.